### PR TITLE
one explicit hypothesis solution for level 8

### PR DESCRIPTION
**Pull request by robinsonkwame**
Uses rewrite directly to prove b=0 and resolves the remaining subgoal simply (`apply h`)

### CODE REVIEW

**Review by adyavanapalli — CHANGES_REQUESTED** [review]
Thank you for your PR! I will approve these changes if you can remove the comments.

**Comment by robinsonkwame** [comment]
Great, I removed the comments and changed h to h₁

File: advanced-addition-world/8 [hunk 1]
```diff
@@ -2,6 +2,6 @@ intro h₁,
 rw ← add_zero a at h₁,
```
Comment (adyavanapalli) [inline advanced-addition-world/8]: Since only one hypothesis is used here, consider changing `h₁` to just `h`.

**Review by adyavanapalli — CHANGES_REQUESTED** [review]

**Comment by robinsonkwame** [comment]
how's that?

**Comment by adyavanapalli** [comment]
Perfect. Thanks for your contribution!

**Pull request merged**